doc/programming/parsing.rst

```diff
@@ -215,7 +215,7 @@ position. Example:
     :show-with: foo.spicy
 
 As you can see, ``x`` receives 6 bytes of input, which it then turns
-into three 16-bit integers.
+into three a-bit integers.
```

Comment:
Why this change?

Reply:
Oops, some accidental doc changes slipped in, will revert.